integration-cli/docker_cli_inspect_test.go

```diff
@@ -142,33 +121,26 @@ func (s *DockerSuite) TestInspectTypeFlagWithInvalidValue(c *check.C) {
 	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "true")
 
 	out, exitCode, err := dockerCmdWithError("inspect", "--type=foobar", "busybox")
-	if exitCode != 0 || err != nil {
-		if !strings.Contains(out, "not a valid value for --type") {
-			c.Fatalf("failed to inspect image: %s, %v", out, err)
-		}
-	}
+	c.Assert(err, checker.NotNil, check.Commentf("%s", exitCode))
+	c.Assert(exitCode, checker.Equals, 1, check.Commentf("%s", err))
```

Comment:
so do you think `exitCode != 0 || err != nil ` with an `Equals false` is the condition we should have in the assert? It's just such a strange case to have. I wish there were a comment beforehand explaining why, but maybe there wasn't a why, it's just what passed.